tools/CMakeLists.txt

```diff
@@ -1,11 +1,107 @@
-list(APPEND TOOLS_SOURCES
+add_library(crashpad_tools STATIC
     tool_support.cc
     tool_support.h
 )
-
-add_library(crashpad_tools STATIC ${TOOLS_SOURCES})
-target_link_libraries(crashpad_tools
-    mini_chromium
+target_link_libraries(crashpad_tools PRIVATE
+    $<BUILD_INTERFACE:crashpad_interface>
 )
 
-# TODO: do we need any tool executables?
+set_property(TARGET crashpad_tools PROPERTY EXPORT_NAME tools)
+add_library(crashpad::tools ALIAS crashpad_tools)
+
+crashpad_install_target(crashpad_tools)
+
+if(CRASHPAD_BUILD_TOOLS)
```

Comment:
nice! these are not really necessary for sentry itself, but I guess you want to package crashpad separately as well?

Reply:
I've added them for completeness sake. I've only build tested the Windows binaries.